roles/openshift_prometheus/defaults/main.yaml

```diff
@@ -0,0 +1,74 @@
+---
+# defaults file for openshift_prometheus
+openshift_prometheus_state: present
+
+openshift_prometheus_namespace: prometheus
+
+openshift_prometheus_replicas: 1
+openshift_prometheus_node_selector: {"region":"infra"}
+
+# images
+openshift_prometheus_image_proxy: "openshift/oauth-proxy:v1.0.0"
+openshift_prometheus_image_prometheus: "openshift/prometheus:v2.0.0-dev"
+openshift_prometheus_image_alertmanager: "openshift/prometheus-alertmanager:dev"
+openshift_prometheus_image_alertbuffer: "ilackarms/message-buffer"
+
+# additional prometheus rules file
+openshift_prometheus_additional_rules_file: null
+
+# All the required exports
```

Comment:
Does prometheus must bound with PV?

Reply:
it is creating pvcs by default but can be overridden if `openshift_prometheus_storage_type` `openshift_prometheus_alertmanager_storage_type` `openshift_prometheus_alertbuffer_storage_type` are set to anything other than `pvc`.
This will become clearer in the nfs PR https://github.com/openshift/openshift-ansible/pull/5459